commercetools-models/src/main/java/io/sphere/sdk/carts/commands/updateactions/SetShippingMethod.java

```diff
@@ -30,15 +29,23 @@ private SetShippingMethod(@Nullable final ResourceIdentifier<ShippingMethod> shi
         return shippingMethod;
     }
 
-    public static SetShippingMethod of(@Nullable final Referenceable<ShippingMethod> shippingMethod) {
+    /**
+     * This method is deprecated, please use {@link SetShippingMethod#of(ResourceIdentifier)}
+     */
+    @Deprecated
+    public static SetShippingMethod ofReferencable(@Nullable final Referenceable<ShippingMethod> shippingMethod) {
```

Comment:
I think this is a typo, the method should be named `ofReferenceable`. The è`is missing 😉

Reply:
Referencable is correct. Interface name is a typo 🙈